samples/CommunityToolkit.Maui.Sample/Models/PopupSize.cs

```diff
@@ -1,14 +1,22 @@
 ﻿namespace CommunityToolkit.Maui.Sample.Models;
 
-static class PopupSizeConstants
+public class PopupSizeConstants
```

Comment:
Probably it should be renamed because it is not Constant. 
Can we make make it a configuration with predefined values?
So if I want to set custom size for all small popups, I need to change the config value only once. Example: https://github.com/xamarin/XamarinCommunityToolkit/blob/6a8c500ce087363cff0dd75514dcdf15de6f407d/src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/Options/ToastOptions.shared.cs#L19